app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2 class="login-heading">Log in</h2>
```

Comment:
- [ ] Good job so you implemented well most of requirement for this project :+1:  Unfortunately  when I run you application I only see a blank page . I am not able to see the form that allow me to register and log in with a combination of email and password and confirm that you devise works as expected . Please fix as it required to make a user to register and login  :+1: 

- [ ] According to the project requirement the  views in devise for the registration and login to match [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)  Please fix it :+1: 

#### You views right now 

![Screenshot_20230201_234612](https://user-images.githubusercontent.com/6615394/216193528-aba33afb-2aa0-4059-a2be-2109e5061b88.png)

Reply:
It renders perfectly on my end as tested multiple times

Reply:
@Shedrack-Sunday Sorry just so you comment  I think you should authenticate all controllers before working with any of them by using
`before_action :authenticate_user!` so that when we run the app we will direct you to the login page :+1: